src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAXIMUM_NUMBER = 100;
+    private final Random random = new Random();
+    private final ColorSupplier randomColor = new ColorSupplier();
```

Comment:
this is not a random color yet :)
```suggestion
    private final ColorSupplier coloerSupplier = new ColorSupplier();
```